src/integration-test/java/com/mongodb/jdbc/integration/testharness/models/TestDataEntry.java

```diff
@@ -7,5 +7,6 @@
     public String db;
     public String collection;
     public List<Map<String, Object>> docs;
+    public List<Map<String, Object>> docsJson;
```

Comment:
Maybe we should document in the yaml, in this file, or in a README what these fields are and how they are used. Just a sentence each saying "`docs` uses standard JSON to represent collection data. This is useful for simple types such as int and string. `docsJson` uses extended JSON to represent collection data. This is useful for complex types such as binData and objectid."

Also another idea: maybe we should name this `docsExtJson`?

Reply:
Added a section to the README with this information and renamed field to `docsExtJson`